src/constants.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2011-2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_CONSTANTS_H
+#define BITCOIN_CONSTANTS_H
+
+#include <cstdint>
+
+/* One gigabyte (GB) in bytes */
+static constexpr uint64_t GB_BYTES{1000000000};
+
+/* One mebibyte (MiB) in bytes */
+static constexpr int32_t MIB_BYTES{1024 * 1024};
```

Comment:
How about introducing this right from the beginning? It's a bit weird that `MIB_BYTES` jumps files with each commit, from `init.cpp` to `validation.h` to its final destination `constants.h`.